### PR TITLE
Update phpmyadmin user and password

### PR DESCRIPTION
**Pull request by Song2017**
- update phpmyadmin user and password to variables DATABASE_USERNAME, DATABASE_PASSWORD
- show phpmyadmin credential error at browser
- default enable remote database

### CODE REVIEW

File: apache-php-mysql/0.4/httpd.conf [hunk 1]
```diff
@@ -41,7 +41,7 @@ CustomLog "/var/log/httpd/access_log" common
         Require all granted
         AllowOverride All
 	    Options FollowSymlinks
-        DirectoryIndex index.php index.html hostingstart.html
+        DirectoryIndex error_localdb.php index.php index.html hostingstart.html
```
Comment (fanjeffrey) [inline apache-php-mysql/0.4/httpd.conf]: If customer uploaded his apps to /home/site/wwwroot without removal of error_localdb.php, what would happen?

File: apache-php-mysql/README.md [hunk 2]
```diff
@@ -73,6 +68,9 @@ On Webssh run the command below to check if the startup logs from entrypoint.sh
 - **Version 0.4**
   1. Update the section Startup Log in README.md.
   2. Create default database - azurelocaldb.(You need set DATABASE_TYPE to **"local"**)
+  3. Security release with **phpMyAdmin**, you need set Log Info on App settings.
```
Comment (fanjeffrey) [inline apache-php-mysql/README.md]: Need refine this. not a good statement.

**Review by fanjeffrey — CHANGES_REQUESTED** [review]

File: apache-php-mysql/0.4/httpd.conf (hunk 1 above)
Comment (Song2017) [reply apache-php-mysql/0.4/httpd.conf]: to ensure app install without effect of error_localdb.php 
 - local mode: remove it with set DATABASE_USERNAME, DATABASE_PASSWORD
 - no local mode(now only remote): remove once it exists

Comment (fanjeffrey) [reply apache-php-mysql/0.4/httpd.conf]: We shouldn't add a DirectoryIndex prior to index.php/index.html here.

Comment (fanjeffrey) [reply apache-php-mysql/0.4/httpd.conf]: That would make user can't see his index.php/html

Comment (Song2017) [reply apache-php-mysql/0.4/httpd.conf]: if there is already a index.php(e.g. wordpress), app will begin to install directly even not enable local database

File: apache-php-mysql/README.md [hunk 3]
```diff
@@ -73,6 +68,10 @@ On Webssh run the command below to check if the startup logs from entrypoint.sh
 - **Version 0.4**
   1. Update the section Startup Log in README.md.
   2. Create default database - azurelocaldb.(You need set DATABASE_TYPE to **"local"**)
+  3. Considering security, please set database authentication info on [*"App settings"*](#deploying-on-azure) when enable **"local"** mode.   
+     Note: Database authentication is also used by phpMyAdmin.
```
Comment (fanjeffrey) [inline apache-php-mysql/README.md]: Note: the credentials below is also used by phpMyAdmin

File: apache-php-mysql/0.4/httpd.conf [hunk 4]
```diff
@@ -41,7 +41,7 @@ CustomLog "/var/log/httpd/access_log" common
         Require all granted
         AllowOverride All
 	    Options FollowSymlinks
-        DirectoryIndex index.php index.html hostingstart.html
+        DirectoryIndex index.php index.html hostingstart.htm hostingstart.html
```
Comment (fanjeffrey) [inline apache-php-mysql/0.4/httpd.conf:44]: Great.

File: apache-php-mysql/README.md (hunk 3 above)
Comment (Song2017) [reply apache-php-mysql/README.md]: got it

**Pull request merged**